backend/api/Models/Property/PropertyFilterModel.cs

```diff
@@ -0,0 +1,315 @@
+        /// </summary>
+        /// <value></value>
+        public double? NELongitude { get; set; }
+
+        /// <summary>
+        /// get/set - South West Latitude.
+        /// </summary>
+        /// <value></value>
+        public double? SWLatitude { get; set; }
+
+        /// <summary>
+        /// get/set - South West Longitude.
+        /// </summary>
+        /// <value></value>
+        public double? SWLongitude { get; set; }
+
+        /// <summary>
+        /// get/set - The property address.
+        /// </summary>
+        /// <value></value>
+        public string Address { get; set; }
+
+        /// <summary>
+        /// get/set - A way to filter both Parcel.LandArea and the Building.BuildingRentableArea.
+        /// </summary>
+        /// <value></value>
+        public float? MinLotArea { get; set; }
+
+        /// <summary>
+        /// get/set - A way to filter both Parcel.LandArea and the Building.BuildingRentableArea.
+        /// </summary>
+        /// <value></value>
+        public float? MaxLotArea { get; set; }
+
+        #region Parcel Filters
+        /// <summary>
+        /// get/set - Building classification Id.
+        /// </summary>
+        /// <value></value>
+        public int? ClassificationId { get; set; }
+
+        /// <summary>
+        /// get/set - Building status Id.
+        /// </summary>
+        /// <value></value>
+        public int? StatusId { get; set; }
+
+        /// <summary>
+        /// get/set - Parcel minimum land area.
+        /// </summary>
+        /// <value></value>
+        public float? MinLandArea { get; set; }
+
+        /// <summary>
+        /// get/set - Parcel maximum land area.
+        /// </summary>
+        /// <value></value>
+        public float? MaxLandArea { get; set; }
+        #endregion
+
+        #region Building Filters
+
+        /// <summary>
+        /// get/set - Building construction type Id.
+        /// </summary>
+        /// <value></value>
+        public int? ConstructionTypeId { get; set; }
+
+        /// <summary>
+        /// get/set - Building predominant use Id.
+        /// </summary>
+        /// <value></value>
+        public int? PredominateUseId { get; set; }
+
+        /// <summary>
+        /// get/set - Building floor count Id.
+        /// </summary>
+        /// <value></value>
+        public int? FloorCount { get; set; }
+
+        /// <summary>
+        /// get/set - Building tenancy.
+        /// </summary>
+        /// <value></value>
+        public string Tenancy { get; set; }
+
+        /// <summary>
+        /// get/set - Building minimum rentable area.
+        /// </summary>
+        /// <value></value>
+        public float? MinRentableArea { get; set; }
+
+        /// <summary>
+        /// get/set - Building maximum rentable area.
+        /// </summary>
+        /// <value></value>
+        public float? MaxRentableArea { get; set; }
+        #endregion
+
+        /// <summary>
```

Comment:
I'm not sure I get what is going on with the next four fields. Doesn't ParcelEvaluations and BuildingEvaluations support the same four fields? (estimatedValue, appraisedValue, assessedValue, netBookValue).

Reply:
Also, I don't understand why these aren't grouped with the other parcel and building specific properties above or in the include sections below.

Reply:
They are grouped, just poorly.  There are two regions that capture the unique fields for Parcels and Buildings.  All others are shared and outside the regions.